packages/minifier-css/minifier.js

```diff
@@ -116,96 +213,77 @@ var rewriteRules = function (rules, mergedCssPath) {
-      var cssUrlRegex = /url\s*\(\s*(['"]?)(.+?)\1\s*\)/gi;
-      while (parts = cssUrlRegex.exec(value)) {
-        oldCssUrl = parts[0];
-        quote = parts[1];
-        resource = url.parse(parts[2]);
-
-
-        // We don't rewrite URLs starting with a protocol definition such as
-        // http, https, or data, or those with network-path references
-        // i.e. //img.domain.com/cat.gif
-        if (resource.protocol !== null || resource.href.startsWith('//') || resource.href.startsWith('#')) {
-          continue;
-        }
+    if (! basePath.match(/^\/?packages\//i)) {
+      basePath = "/";
+    }
 
-        // Rewrite relative paths (that refers to the internal application tree)
-        // to absolute paths (addressable from the public build).
-        if (isRelative(resource.path)) {
-          absolutePath = pathJoin(basePath, resource.path);
-        } else {
-          absolutePath = resource.path;
-        }
-        
-        if (resource.hash) {
-          absolutePath += resource.hash;
-        }
+    let value = rule.value;
+
+    // Match css values containing some functional calls to `url(URI)` where
+    // URI is optionally quoted.
+    // Note that a css value can contains other elements, for instance:
+    //   background: top center url("background.png") black;
+    // or even multiple url(), for instance for multiple backgrounds.
+    var cssUrlRegex = /url\s*\(\s*(['"]?)(.+?)\1\s*\)/gi;
+    let parts;
+    while (parts = cssUrlRegex.exec(value)) {
+      const oldCssUrl = parts[0];
+      const quote = parts[1];
+      const resource = url.parse(parts[2]);
 
-        // We used to finish the rewriting process at the absolute path step
-        // above. But it didn't work in case the Meteor application was deployed
-        // under a sub-path (eg `ROOT_URL=http://localhost:3000/myapp meteor`)
-        // in which case the resources linked in the merged CSS file would miss
-        // the `myapp/` prefix. Since this path prefix is only known at launch
-        // time (rather than build time) we can't use absolute paths to link
-        // resources in the generated CSS.
-        //
-        // Instead we transform absolute paths to make them relative to the
-        // merged CSS, leaving to the browser the responsibility to calculate
-        // the final resource links (by adding the application deployment
-        // prefix, here `myapp/`, if applicable).
-        relativeToMergedCss = pathRelative(mergedCssPath, absolutePath);
-        newCssUrl = "url(" + quote + relativeToMergedCss + quote + ")";
-        value = value.replace(oldCssUrl, newCssUrl);
+      // We don't rewrite URLs starting with a protocol definition such as
+      // http, https, or data, or those with network-path references
+      // i.e. //img.domain.com/cat.gif
+      if (resource.protocol !== null
+          || resource.href.startsWith('//')
+          || resource.href.startsWith('#')) {
```

Comment:
We usually put the `||` or `&&` operator at the end of the line, so the conditions line up:
```js
if (resource.protocol !== null ||
    resource.href.startsWith('//') ||
    resource.href.startsWith('#')) {
```

Reply:
Sounds good - I'll adjust, thanks!